examples/go/tagging/.gitignore

```diff
@@ -0,0 +1,3 @@
+template
+build
+vcconfig.toml
```

Comment:
`vcconfig.toml` should be included in the repo as its required. For local development and to not expose sensitive data perhaps use a different file name and exclude that.